zul/src/org/zkoss/zul/Frozen.java

```diff
@@ -100,6 +101,33 @@ public int getRows() {
 		return 0;
 	}
 
+	/**
+	 * Only available in ZK EE.

```

Comment:
Shouldn't put to the first line or Javadoc will pick this as an one-line description